src/index.css

```diff
@@ -95,4 +95,26 @@
 .style-title-page{
   font-weight: bold;
   font-size: 30px;
+}
+
+.btn-table{
+  background-color: #dbb82d;
+  border: none;
+  cursor: pointer;
+  font-size: 16px;
+  color: white;
+}
+
+.mr{
+  margin-right: 5px;
+}
+
+.btn:first-child:hover{
+  background-color:#dbb82d;
+  color: white;
+}
+
+:not(.btn-check)+.btn:hover{
```

Comment:
I think we can improve this selector. Creating a new class

Reply:
I try to do it with a new class and it does not take the change.